ansible/roles/software/ckan/inventory/molecule/default/tests/test_default.py

```diff
@@ -37,3 +37,22 @@ def test_apache2(host):
 
     assert apache2.is_enabled
     assert apache2.is_running
+
+
+def test_beaker_cache_cleanup(host):
+    script = host.file('/usr/local/bin/beaker-cache-cleanup.sh')
+    supervisor_conf = host.file(
```

Comment:
These should be separate tests to be consistent with how we write other tests, but :man_shrugging:

Reply:
ok, will separate the tests in another PR. let us get this one thru for now.